lang/python/2to3/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=2to3
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=COPYRIGHT
+
+PKG_HOST_ONLY:=1
+BUILDONLY:=1
+HOST_BUILD_DEPENDS:=python3/host
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=lib2to3
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/2to3
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=2to3 binary using python3
+  DEPENDS:= \
+	+python3-lib2to3 \
+	+python3-light
+  URL:=https://salsa.debian.org/cpython-team/python3-defaults
+endef
+
+define Package/2to3/description
+  2to3 is a Python program that reads Python 2.x source code and applies a
+  series of fixers to transform it into valid Python 3.x code. The standard
+  library contains a rich set of fixers that will handle almost all code. 2to3
+  supporting library lib2to3 is, however, a flexible and generic library, so it
+  is possible to write your own fixers for 2to3. lib2to3 could also be adapted
+  to custom applications in which Python code needs to be edited automatically.
+endef
+
+define Host/Install
+	$(INSTALL_DIR) $(STAGING_DIR_HOST)/bin
+	$(INSTALL_BIN) ./files/2to3.py $(STAGING_DIR_HOST)/bin/2to3
```

Comment:
Please install to `$(1)/bin/2to3` - `$(1)` [will be](https://github.com/openwrt/openwrt/blob/9ba6ee4e25da40f5c7dab5f261e8eace8111fe9d/include/host-build.mk#L168) `$(HOST_BUILD_PREFIX)`, and `$(HOST_BUILD_PREFIX)` [will be](https://github.com/openwrt/openwrt/blob/9ba6ee4e25da40f5c7dab5f261e8eace8111fe9d/include/host-build.mk#L27) `$(STAGING_DIR_HOSTPKG)`.

Reply:
fixed